doc/manual/src/command-ref/files/channels.md

```diff
@@ -1,26 +1,41 @@
 ## Channels
 
-A directory containing symlinks to Nix channels, managed by [`nix-channel`]:
-
-- `$XDG_STATE_HOME/nix/profiles/channels` for regular users
-- `$NIX_STATE_DIR/profiles/per-user/root/channels` for `root`
-
-[`nix-channel`] uses a [profile](@docroot@/command-ref/files/profiles.md) to store channels.
-This profile contains symlinks to the contents of those channels.
+Channels are a mechanism for obtaining Nix expressions over the web.
+
+The moving parts of channels are:
+- The official channels listed at <https://nixos.org/channels>
+- The user-specific list of [subscribed channels](#subscribed-channels)
+- The [downloaded channel contents](#channel-contents)
+- The [Nix expression search path](@docroot@/command-ref/conf-file.md#conf-nix-path)
+
+> **Note**
+>
+> The state of a subscribed channel is external to the Nix expressions relying on it.
+> This may limit reproducibility.
+>
+> Dependencies on other Nix expressions can be declared explicitly with:
+> - [`fetchurl`](@docroot@/language/builtins.md#builtins-fetchurl), [`fetchTarball`](@docroot@/language/builtins.md#builtins-fetchTarball), or [`fetchGit`](@docroot@/language/builtins.md#builtins-fetchGit) in Nix expressions
+> - the [`-I` option](@docroot@/command-ref/opt-common.md#opt-I) in command line invocations for explicitly determining the value of [lookup paths](@docroot@/language/constructs/lookup-path.md)
 
 ## Subscribed channels
 
-The list of subscribed channels is stored in
+The list of subscribed channels is stored in a file:
 
 - `~/.nix-channels`
-- `$XDG_STATE_HOME/nix/channels` if [`use-xdg-base-directories`] is set to `true`
+- `$XDG_STATE_HOME/nix/channels` if [`use-xdg-base-directories`](@docroot@/command-ref/conf-file.md#conf-use-xdg-base-directories) is set to `true`
 
-in the following format:
+Each line maps a channel name to a URL in the following format:
 
 ```
 <url> <name>
-...
 ```
 
-[`nix-channel`]: @docroot@/command-ref/nix-channel.md
-[`use-xdg-base-directories`]: @docroot@/command-ref/conf-file.md#conf-use-xdg-base-directories
+## Channels contents
+
+The [`nix-channel`](@docroot@/command-ref/nix-channel.md) command uses a [profile](@docroot@/command-ref/files/profiles.md) to keep track of channels:
+
+- `$XDG_STATE_HOME/nix/profiles/channels` for regular users
+- `$NIX_STATE_DIR/profiles/per-user/root/channels` for `root`
```

Comment:
```suggestion
- profile `$XDG_STATE_HOME/nix/profiles/channels` for regular users
- profile `$NIX_STATE_DIR/profiles/per-user/root/channels` for `root`
```

Reply:
I'm not sure what this is supposed to achieve